webview/platforms/cocoa.py

```diff
@@ -275,10 +275,8 @@ def mouseDragged_(self, event):
 
             super(BrowserView.WebKitHost, self).mouseDown_(event)
 
-        def rightMouseDown_(self, event):
-            i = BrowserView.get_instance('webkit', self)
-            if _debug['mode']:
-                super(BrowserView.WebKitHost, self).rightMouseDown_(event)
+        def willOpenMenu_withEvent_(self, menu, event):
+            menu.removeAllItems()
```

Comment:
Please put debug mode back. Right click menu is visible in debug mode by design.

Reply:
Can I put the debug check in the new willOpenMenu_withEvent_ rather than adding back the rightMouseDown_, since that would revert back to blocking javascript events.  In other words (with a "not"), just:

        def willOpenMenu_withEvent_(self, menu, event):
            if not _debug['mode']:
                menu.removeAllItems()

Reply:
...If that sounds good to you, the latest commit has that.